exercises/allergies/uAllergyTests.pas

```diff
@@ -10,10 +10,11 @@ interface
   AllergyTests = class(TObject) 
   public
     [Test]
+//  [Ignore('Comment the "[Ignore]" statement to run the test')]
     procedure No_allergies_means_not_allergic;
```

Comment:
Somehow the recursion is alluring.  `Comment this statement to run the test` would probably be OK... and yet, somehow this recursion is alluring.